ci/jenkins/test-vmc.sh

```diff
@@ -396,12 +397,8 @@ function run_integration {
     VM_IP=$(govc vm.ip ${VM_NAME}) # wait for VM to be on
 
     set -x
-    echo "===== Run Integration test ====="
-    if [[ ${DOCKER_REGISTRY} != "" ]]; then
-        docker pull ${DOCKER_REGISTRY}/antrea/openvswitch:2.14.0
-        docker tag ${DOCKER_REGISTRY}/antrea/openvswitch:2.14.0 antrea/openvswitch:2.14.0
-    fi
```

Comment:
@lzhecheng I think this code was wrong. It was pulling the `antrea/openvswitch:2.14.0` image on the Jenkins worker node and not on the VM running the integration tests.

Reply:
Yes, you are right. Thanks for deleting these.
Besides, integration test timeouts. Maybe we should extend the length.
https://github.com/vmware-tanzu/antrea/blob/main/ci/jenkins/jobs/projects.yaml#L169

Reply:
Since the job change will not take place until this PR is merged, you need to manually update timeout on Jenkins.